service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseUrlService.java

```diff
@@ -8,6 +9,7 @@
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
 @Service
+@Getter
```

Comment:
not an issue but in theory is it okay to have Getter on a service? but then i am not sure if this class is a service 🤷

Reply:
hmm, good point. I added this here because i wanted to access xuiBaseUrl but i think i could add this property into the class I need it and remove the dependency on this service. So will remove. and don't think it should be here :)